setuptools/_vendor/tomli/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""A lil' TOML parser."""
+
+__all__ = ("loads", "load", "TOMLDecodeError")
+__version__ = "1.2.3"  # DO NOT EDIT THIS LINE MANUALLY. LET bump2version UTILITY DO IT
+
+from tomli._parser import TOMLDecodeError, load, loads
```

Comment:
Needs to be a relative import?

Reply:
Thank you very much for the review @blink1073.

Relative imports are not necessary :)
Thanks to the MetaPathFinder implemented in `setuptools.extern`, we don't have to rewrite this.

Reply:
Gotcha, thanks

Reply:
Sorry @blink1073, I completely misinterpreted the code in `setuptools.extern`. I was just reading it again and I am no longer sure this is handled...

I will rewrite this imports anyway just in case.